pkg/providers/instancetype/suite_test.go

```diff
@@ -167,10 +167,9 @@ var _ = Describe("Instance Types", func() {
 			v1alpha1.LabelInstanceGPUCount:                     "1",
 			v1alpha1.LabelInstanceGPUMemory:                    "16384",
 			v1alpha1.LabelInstanceLocalNVME:                    "900",
-			v1alpha1.LabelInstanceAcceleratorName:              "t4",
-			v1alpha1.LabelInstanceAcceleratorManufacturer:      "nvidia",
+			v1alpha1.LabelInstanceAcceleratorName:              "inferentia",
+			v1alpha1.LabelInstanceAcceleratorManufacturer:      "aws",
```

Comment:
Does this actually change anything? Shouldn't this work identically to before?

Reply:
Yes, it does but "nvidia" is no longer a valid accelerator manufacturer since we aren't propagating that label for GPUs with this change